packages/datadog-plugin-aws-sdk/src/services/bedrockruntime.js

```diff
@@ -0,0 +1,207 @@
+'use strict'
+
+const BaseAwsSdkPlugin = require('../base')
+const log = require('../../../dd-trace/src/log')
+
+const _AI21 = 'ai21'
+const _AMAZON = 'amazon'
+const _ANTHROPIC = 'anthropic'
+const _COHERE = 'cohere'
+const _META = 'meta'
+const _STABILITY = 'stability'
+
+class BedrockRuntime extends BaseAwsSdkPlugin {
+  static get id () { return 'bedrock runtime' }
+
+  generateTags (params, operation, response) {
+    let tags = {}
+    let modelName = ''
+    let modelProvider = ''
+    const modelMeta = params.modelId.split('.')
+    if (modelMeta.length === 2) {
+      [modelProvider, modelName] = modelMeta
+      modelProvider = modelProvider.toUpperCase()
```

Comment:
any reason to have it uppercased here? i think it fails conditions when checking agains `_AMAZON`, `_AI21`, etc. as those constants' values are all lowercase

Reply:
if needed, maybe we can force `toLowerCase`. We can definitely keep it uppercase, but it has to be consistent with the casing of the constants above (which right now are lowercase)